openmc/executor.py

```diff
@@ -48,14 +51,34 @@ def working_directory(self, working_directory):
 
         self._working_directory = working_directory
 
-
     def plot_geometry(self, output=True):
-        self._run_openmc('openmc -p', output)
+        """Run OpenMC in plotting mode"""
 
+        self._run_openmc('openmc -p', output)
 
     def run_simulation(self, particles=None, threads=None,
                        geometry_debug=False, restart_file=None,
                        tracks=False, mpi_procs=1, output=True):
+        """Run an OpenMC simulation.
+
+        Parameters
```

Comment:
Will sphinx pick up on the default keyword parameter values when it generates the documentation? Will the default values be included only in the method signature, or will they also be itemized next to each parameter in the subsequent text?